playwright.config.ts

```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   testDir: "./playwright-e2e",
   /* Run tests in files in parallel */
   fullyParallel: true,
-  timeout: 180000,
-  expect: { timeout: 180000 },
+  timeout: 250000,
```

Comment:
is timeout needed to be raised due to allocation proposal screens only?

Reply:
@GauravChawlaMOJ this was increased because Charles connection is so slow with AAT it can't handle less then this time out. It will likely help our nightly pipeline too as AAT is so problematic with performance.